CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 23.1.6 [#688](https://github.com/openfisca/openfisca-core/pull/688)
```

Comment:
Why jump version `23.1.5`? 🤔

Reply:
Something told me `23.1.5` would be merged before ;)